tests/end-to-end/event/data-provider-duplicate-key.phpt

```diff
@@ -19,7 +19,7 @@ Data Provider Method Finished for PHPUnit\TestFixture\Event\DataProviderDuplicat
 - PHPUnit\TestFixture\Event\DataProviderDuplicateKeyTest::provider
 Test Triggered PHPUnit Error (PHPUnit\TestFixture\Event\DataProviderDuplicateKeyTest::testSomething)
 The data provider specified for PHPUnit\TestFixture\Event\DataProviderDuplicateKeyTest::testSomething is invalid
-The key "key" has already been defined by a previous data provider
+The key "key" has already been defined by provider PHPUnit\TestFixture\Event\DataProviderDuplicateKeyTest::provider
```

Comment:
I went with a fully qualified method reference in the error messages, as these can easily be navigated to by e.g. PHPStorm.